qiskit/circuit/parametertable.py

```diff
@@ -165,7 +166,16 @@ def get_names(self):
         Returns:
             set: A set of all the names in the parameter table
         """
-        return self._names
+        return self._names.keys()
+
+    def parameter_from_name(self, name: str, default: typing.Any = None):
+        """Get a :class:`.Parameter` with references in this table by its string name, or return the
+        default if not present.
+
+        Args:
+            name: the name of the :class:`.Parameter`
+            default: the object that should be returned if the parameter is missing."""
```

Comment:
Just a suggestion for more consistent formatting, feel free to ignore or adapt!
```suggestion
        """Get a :class:`.Parameter` with references in this table by its string name.
        
         If the parameter is not present, returns the ``default`` value.

        Args:
            name: The name of the :class:`.Parameter`
            default: The object that should be returned if the parameter is missing.
        """
```